layouts/shortcodes/issues-security.html

```diff
@@ -0,0 +1,19 @@
+<table class="security-cves">
+  <caption style="text-align:center">Table: Official Kubernetes CVE List</caption>
+  <thead>
+  <tr>
+    <th>{{ T "cve_id" }}</th>
+    <th>{{ T "cve_issue_summary"}}</th>
+    <th>{{ T "cve_issue_details" }}</th>
+  </tr>
+  </thead>
+  <tbody>
+  {{ range $issues := getJSON "static/security/official-cve-feed.json" }}
```

Comment:
This could be a `getJSON` doing a HTTP (well, HTTPS) fetch. Hugo will cache the result.